### PR TITLE
CLDR-17316 cldr-apps-webdriver Java formatting with spotless

### PR DESCRIPTION
**Pull request by btangmu**
-Add toggleOffOn in pom.xml (the cldr project does not have that)

-Add spotless:on and spotless:off comments in SurveyDriverData.java, replacing prettier-ignore

-Other changes are only formatting results of running mvn spotless:apply

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -58,6 +58,7 @@
             <ratchetFrom>NONE</ratchetFrom>
             <!-- define a language-specific format -->
             <java>
+              <toggleOffOn />
```
Comment (btangmu) [inline pom.xml:61]: @srl295 I'm a little surprised that cldr didn't need this, which enables spotless:on/off comments for disabling formatting of a few long arrays... probably those arrays should be implemented differently anyway but I think this is better than one item per line...

Comment (srl295) [reply pom.xml:61]: CLDR probably needs this but didn't know about it. we just let spotless do what it wants to do. want to file an issue and/or put on the infra agenda?

Comment (btangmu) [reply pom.xml:61]: I'll put it on the agenda. I don't remember any specific instances of anyone being unhappy with the spotless results. I just vaguely recall such issues when various formatting options were being considered.

Comment (btangmu) [reply pom.xml:61]: anyway is this PR okay?

**Review by srl295 — APPROVED** [review]

**Pull request merged**